roles/logging/templates/etc/rsyslog.d/60-remote-syslog.conf

```diff
@@ -0,0 +1,17 @@
+$WorkDirectory {{ logging.syslog_forwarding.work_directory }}
+
+$ActionResumeRetryCount {{ logging.syslog_forwarding.action_resume.retry_count }}
```

Comment:
In the new PR for 3.1.X I changed it iterate through a list. https://github.com/blueboxgroup/ursula/pull/2367. Please let me know what you think. Thanks.